math/math_helper.go

```diff
@@ -213,14 +213,21 @@ func AddInt(i ...int) (int, error) {
 	return sum, nil
 }
 
+// Wei is the smallest unit of Ether, and sent in the payload value of the execution engine API call.
+// represented as a pointer to a bigInt.
```

Comment:
```suggestion
// Wei is the smallest unit of Ether, represented as a pointer to a bigInt.
```
I don't think a type definition should specify a particular use case, although it's a nitpick really.